src/main/java/tk/shanebee/hg/listeners/GameListener.java

```diff
@@ -447,6 +456,29 @@ private void onInteract(PlayerInteractEvent event) {
 		}
 	}
 
+	@EventHandler
+	private void onInventoryClick(InventoryClickEvent e){
+		if (e.getWhoClicked() instanceof Player){
+			Player player = (Player) e.getWhoClicked();
+			InventoryAction action = e.getAction(); //might not be needed
+			Game g1 = playerManager.getPlayerData(player).getGame();
+			Status status = g1.getGameArenaData().getStatus();
+			if (status != Status.RUNNING && status != Status.BEGINNING){
+				if (e.getClickedInventory().contains(Material.RED_BED)){
+					//Bukkit.dispatchCommand(player, "hg leave");
+					g1.getGamePlayerData().leave(player,false);
+				}
+				else if (e.getClickedInventory().contains(Material.NETHER_STAR)){
```

Comment:
Spacing and remove the commented out line
```suggestion
			if (status != Status.RUNNING && status != Status.BEGINNING) {
				if (e.getClickedInventory().contains(Material.RED_BED)) {
					g1.getGamePlayerData().leave(player,false);
				} else if (e.getClickedInventory().contains(Material.NETHER_STAR)) {
```